examples/TestAppUwp/MainPage.xaml.cs

```diff
@@ -262,47 +75,54 @@ public static string GetDeviceName()
 
         public MainPage()
         {
+            RestoreSettings();
+
             this.InitializeComponent();
 
-            muteLocalVideoStroke.Visibility = Visibility.Collapsed;
-            muteLocalAudioStroke.Visibility = Visibility.Collapsed;
+            // Insert items manually into navigation menu. It doesn't seem that there is a way
+            // to force selecting a menu item when using data binding, so selecting the first
+            // page below would crash (empty MenuItems list if assigning MenuItemsSource).
+            //foreach (var catBase in Categories)
+            //{
+            //    if (catBase is Category cat)
+            //    {
+            //        var item = new NavigationViewItem()
+            //        {
+            //            Content = cat.Name,
+            //            Icon = new SymbolIcon() { Symbol = cat.Glyph },
+            //            DataContext = catBase
+            //        };
+            //        AutomationProperties.SetName(item, cat.Name);
+            //        navigationView.MenuItems.Add(item);
+            //    }
+            //    else if (catBase is Header headerCat)
+            //    {
+            //        var item = new NavigationViewItemHeader()
+            //        {
+            //            Name = headerCat.Name,
+            //            DataContext = catBase
+            //        };
+            //        AutomationProperties.SetName(item, headerCat.Name);
+            //        navigationView.MenuItems.Add(item);
+            //    }
+            //    else if (catBase is Separator sepCat)
+            //    {
+            //        navigationView.MenuItems.Add(new NavigationViewItemSeparator());
+            //    }
+            //}
 
-            // This will be enabled once the signaling is initialized and ready to send data
-            createOfferButton.IsEnabled = false;
+            // Open the "Signaling" page by default
+            //navigationView.MenuItemsSource = Categories;
+            rootFrame.Navigate((Categories[0] as Category).PageType);
```

Comment:
nit: change to `rootFrame.Navigate(typeof(SignalingPage))`?

Reply:
No the comment is wrong but the code is right, that's the core navigation code which actually switch to the correct page (ignoring the bug when you click twice on the same page, first goes correctly, second move to last page for some reason, but let's ignore for now).

Reply:
Fix the comment then! 😄

Reply:
I'm sorry you're right it does navigate to the signaling page. The advantage of that code is we don't need to maintain 2 different places where we have the type of the page, instead `Categories` is the only source of truth. This makes also the code (but not the comment) robust to changes if we prepend a new page at index #0, it will still open the first one.

Reply:
Fixed.